lib/vrl/stdlib/src/parse_syslog.rs

```diff
@@ -22,10 +22,21 @@ impl Function for ParseSyslog {
     fn examples(&self) -> &'static [Example] {
         &[Example {
             title: "parse syslog",
-            source: r#"encode_json(parse_syslog!(s'<13>1 2020-03-13T20:45:38.119Z dynamicwireless.name non 2426 ID931 [exampleSDID@32473 iut="3" eventSource= "Application" eventID="1011"] Try to override the THX port, maybe it will reboot the neural interface!'))"#,
-            result: Ok(
-                r#"s'{"appname":"non","exampleSDID@32473.eventID":"1011","exampleSDID@32473.eventSource":"Application","exampleSDID@32473.iut":"3","facility":"user","hostname":"dynamicwireless.name","message":"Try to override the THX port, maybe it will reboot the neural interface!","msgid":"ID931","procid":2426,"severity":"notice","timestamp":"2020-03-13T20:45:38.119+00:00","version":1}'"#,
-            ),
+            source: r#"parse_syslog!(s'<13>1 2020-03-13T20:45:38.119Z dynamicwireless.name non 2426 ID931 [exampleSDID@32473 iut="3" eventSource= "Application" eventID="1011"] Try to override the THX port, maybe it will reboot the neural interface!')"#,
+            result: Ok(r#"{
+                    "appname":"non",
+                    "exampleSDID@32473.eventID":"1011",
+                    "exampleSDID@32473.eventSource":"Application",
+                    "exampleSDID@32473.iut":"3",
+                    "facility":"user",
+                    "hostname":"dynamicwireless.name",
+                    "message":"Try to override the THX port, maybe it will reboot the neural interface!",
+                    "msgid":"ID931",
+                    "procid":2426,
+                    "severity":"notice",
+                    "timestamp":"2020-03-13T20:45:38.119Z",
+                    "version":1
+                }"#),
```

Comment:
```suggestion
            result: Ok(r#"{
                "appname":"non",
                "exampleSDID@32473.eventID":"1011",
                "exampleSDID@32473.eventSource":"Application",
                "exampleSDID@32473.iut":"3",
                "facility":"user",
                "hostname":"dynamicwireless.name",
                "message":"Try to override the THX port, maybe it will reboot the neural interface!",
                "msgid":"ID931",
                "procid":2426,
                "severity":"notice",
                "timestamp":"2020-03-13T20:45:38.119Z",
                "version":1
            }"#),
```